deploy/zookeeper/quick-start-volume-emptyDir/zookeeper-3-nodes-create.sh

```diff
@@ -5,5 +5,5 @@ ZK_NAMESPACE="${ZK_NAMESPACE:-zoo3ns}"
 CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 kubectl create namespace "${ZK_NAMESPACE}"
-kubectl --namespace="${ZK_NAMESPACE}" apply -f "${CUR_DIR}/zookeeper-3-nodes.yaml"
+cat "${CUR_DIR}/zookeeper-3-nodes.yaml" | kubectl --namespace="${ZK_NAMESPACE}" apply -f -
```

Comment:
This change is not needed. kubectl apply -f <filename.yaml> is a kubernetes way.